.github/workflows/cc-payload-after-push-amd64.yaml

```diff
@@ -27,6 +29,11 @@ jobs:
           - cc-tdx-qemu
           - cc-tdx-td-shim
           - cc-tdx-tdvf
+        include:
+          - measured_rootfs: yes
+            asset:
+              - cc-kernel
+              - cc-tdx-kernel
```

Comment:
It doesn't change the outcome but the cc-tdx-kernel is already built with `MEASURED_ROOTFS=yes` (it is explicitly set in https://github.com/kata-containers/kata-containers/blob/CCv0/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh#L763). I am in favor of removing that export from kata-deploy-binaries.sh (can be in a follow up PR).

Reply:
I will do in a follow-up PR, after making sure the image is actually generated as we want.